data/morphology/unimorph_languages.json

```diff
@@ -1,4 +1,20 @@
 {
-    "eng": "http://raw.githubusercontent.com/unimorph/eng/master/eng",
-    "ger": "http://raw.githubusercontent.com/unimorph/deu/master/deu"
-}
+        "ady": "https://raw.githubusercontent.com/unimorph/ady/master/ady",
```

Comment:
This list is partial. Do you want to complete it before we submit? I suggested earlier that you could just do it for the 21 challenge languages initially, so we can get this in production.

Reply:
I had a list of 21 lgs and since a few don't have unimorph github links, where should I pick lgs from to have 21 language files to start with?

Reply:
Marking this resolved since we'll expand from there.